### PR TITLE
PS2-3, Bugfixes

### PR DESCRIPTION
**Pull request by MrRozger**

### CODE REVIEW

File: SzczepionkaAPI/src/main/java/com/szczepionka/configuration/ApplicationConfig.java [hunk 1]
```diff
@@ -31,4 +32,9 @@ public DistanceCalculator distanceCalculator() {
     public ZipcodeConverter zipcodeConverter() {
         return new ZipcodeConverter();
     }
+
+    @Bean
+    public PatientLinkGenerator patientLinkGenerator() {
+        return new PatientLinkGenerator();
```
Comment (aksuz) [inline SzczepionkaAPI/src/main/java/com/szczepionka/configuration/ApplicationConfig.java:38]: this is a static class, why have you added it as a @Bean? Is it neccessary?

File: SzczepionkaAPI/src/test/java/com/szczepionka/service/AppointmentServiceScenarios.java [hunk 2]
```diff
@@ -32,18 +33,20 @@
...
+    private EmailService emailServiceStub;
 
     @BeforeAll
     void setUp() {
         appointmentRepositoryStub = mock(AppointmentRepository.class);
         patientServiceStub = mock(PatientService.class);
         vaccinationLocationsFetcherStub = mock(VaccinationLocationsFetcher.class);
         ModelMapper modelMapper = new ModelMapper();
-        appointmentService = new AppointmentService(appointmentRepositoryStub, patientServiceStub, vaccinationLocationsFetcherStub, modelMapper);
+        appointmentService = new AppointmentService(appointmentRepositoryStub, patientServiceStub, vaccinationLocationsFetcherStub, modelMapper,emailServiceStub);
```
Comment (xNIGHTHAWKx) [inline SzczepionkaAPI/src/test/java/com/szczepionka/service/AppointmentServiceScenarios.java:45]: Missing space.

File: SzczepionkaAPI/src/main/java/com/szczepionka/configuration/ApplicationConfig.java (hunk 1 above)
Comment (MrRozger) [reply SzczepionkaAPI/src/main/java/com/szczepionka/configuration/ApplicationConfig.java:38]: fixed

File: SzczepionkaAPI/src/test/java/com/szczepionka/service/AppointmentServiceScenarios.java (hunk 2 above)
Comment (MrRozger) [reply SzczepionkaAPI/src/test/java/com/szczepionka/service/AppointmentServiceScenarios.java:45]: fixed

**Pull request merged**